templates/markdown/.partials/schema-prop.md

```diff
@@ -1,14 +1,20 @@
 {% macro schemaProp(prop, propName, required=false, path='') %}
 <tr>
   <td>{{ path | tree }}{{ propName }} {% if required %}<strong>(required)</strong>{% endif %}</td>
-  <td>{{ prop | log }}{{ prop.type() }}{%- if prop.anyOf() -%}anyOf{%- endif -%}{%- if prop.oneOf() %}oneOf{%- endif -%}{%- if prop.items().type %}({{prop.items().type()}}){%- endif -%}</td>
+  <td>{{ prop | log }}{{ prop.type() }}{%- if prop.anyOf() -%}anyOf{%- endif -%}{%- if prop.allOf() -%}allOf{%- endif -%}{%- if prop.oneOf() %}oneOf{%- endif -%}{%- if prop.items().type %}({{prop.items().type()}}){%- endif -%}</td>
```

Comment:
Would you mind getting rid of `{{ prop | log }}` at the beginning of this line? At some point, I left it there by mistake.